src/logger.rs

```diff
@@ -35,11 +44,11 @@ impl Logger {
     /// If `inner` is `None`
     ///     - In dev mode, use a std output
     ///     - In release mode, use a storage specific file with name `LOG`
-    pub fn new<S: Storage>(
+    pub fn new<S: Storage + Clone + 'static>(
         inner: Option<slog::Logger>,
         level: LevelFilter,
-        storage: &S,
-        db_path: &str,
+        storage: S,
+        db_path: String,
```

Comment:
`db_path` could be a `&'static str` I think

Reply:
I will try